docker-entrypoint.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
-cd rems
```

Comment:
rahti might need this. @vvaltia knows better

Reply:
I think this is fine. It was an unnecessary command failing every time